addons/fleet/views/fleet_vehicle_cost_views.xml

```diff
@@ -274,49 +263,30 @@
-                <field name="activity_ids"/>
-                <field name="activity_state"/>
-                <field name="purchaser_id"/>
-                <field name="vendor_id"/>
-                <field name="amount"/>
                 <progressbar field="activity_state" colors='{"planned": "success", "today": "warning", "overdue": "danger"}'/>
                 <templates>
-                    <t t-name="kanban-box">
-                        <div t-attf-class="oe_kanban_global_click container" class="o_kanban_record_has_image_fill">
-                            <div class="oe_kanban_details">
-                                <div class="o_kanban_record_top">
-                                    <img t-att-src="kanban_image('fleet.vehicle', 'image_128', record.vehicle_id.raw_value)" t-att-alt="record.vehicle_id.value" class="o_image_24_cover float-start"/>
-                                    <div class="o_kanban_record_headings ps-2 pe-2">
-                                        <div class="text-truncate">
-                                            <strong class="o_kanban_record_title">
-                                                <field name="vehicle_id"/>
-                                                <span t-attf-class="float-end badge #{['todo', 'running'].indexOf(record.state.raw_value) > -1 ? 'text-bg-secondary' : ['cancelled'].indexOf(record.state.raw_value) > -1 ? 'text-bg-danger' : 'text-bg-success'}">
-                                                    <field name="state"/>
-                                                </span>
-                                            </strong>
-                                        </div>
-                                        <div class="text-truncate">
-                                            <em><field name="service_type_id"/></em>
-                                        </div>
-                                    </div>
-                                </div>
-                                <div class="text-truncate">
-                                    <field name="purchaser_id"/>
-                                    <span class="float-end"><field name="date"/></span>
-                                </div>
+                    <t t-name="kanban-card">
+                        <div>
+                            <field name="vehicle_id" widget="image" options="{'preview_image': 'image_128'}" t-att-alt="record.vehicle_id.value" style="width:40px;" class="float-start me-2"/>
+                            <div class="pe-2 text-truncate">
+                                <field class="fw-bolder" name="vehicle_id"/>
+                                <span t-attf-class="float-end badge #{['todo', 'running'].indexOf(record.state.raw_value) > -1 ? 'text-bg-secondary' : ['cancelled'].indexOf(record.state.raw_value) > -1 ? 'text-bg-danger' : 'text-bg-success'}">
+                                    <field name="state"/>
+                                </span>
```

Comment:
not necessary to wrap the field in a span (you can define the class directly on the field)

Reply:
not working in field :thinking:

Reply:
@omra-odoo It's not that easy, so I suggest you keep it as is.